docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+Strapi is back-end and front-end agnostic. Many different front-ends and databases can (and will) be used. Also, to front-ends and back-ends, many packages, tools, and third-party service providers are often used with Strapi. Referring to the projects and services correctly is essential. If you use incorrect terms to refer to external services, users could get confused when further research outside the Strapi docs is required.
+
+Common projects and how to refer to them:
+
+| Incorrect Usage     | Correct usage |                   Example                   |
+| ------------------- | :-----------: | :-----------------------------------------: |
+| mariadb, mariaDB    |    MariaDB    |    The power and flexibility of MariaDB.    |
+| mongo, mongodb      |    MongoDB    |     MongoDB is a document database, ...     |
+| mysql, MYSQL        |     MySQL     |  You can discuss issues with MySQL users.   |
+| Node, NODE, node.js |    Node.js    |   The Node.js Twitter account is @nodejs.   |
+| npm, NPM            |      npm      | The current stable version of npm is here.  |
+| NPX, Npx, npx       |      npx      | Run packages without downloading using npx. |
+| PM2, pm2, Pm2       |      pm2      | Guys just installed pm2 on my live server.  |
+| postgressql         |  PostgreSQL   |  Want to learn more about PostgreSQL, ...   |
+| React, react.js     |     React     | React allows you to interface with other... |
+| SqLite, sqlite      |    SQLite     |       SQLite is in the public domain.       |
+| Strapi, Strapi.js   |    Strapi     |   Get ready to get Strapi up and running.   |
+| Vue, VUE, Vue.js    | Vue or Vue.js |  My favorite component library for Vue...   |
+| yarn, YARN, Yarn    |     Yarn      | You can use Yarn to keep it all up to date. |
+
+---
+
+#### Using images
+
+If you use images in the documentation, it is a best practice to include both HTML **alt text** and **title text**. The **alt text** appears in place of a broken link, and the **title text** appears when someone hovers over the image. Both are used to help with _accessibility_.
+
+Markdown example,
+
+```
+![alt-text](/src/of/the/image.png) 'title of image')
+```
+
+The following example generates the following logo.
+
+```
+![The Strapi Logo](../assets/contribution-guide/contribute-with-docs/strapi-logo.png 'Strapi Logo')
+```
+
+#### The Strapi Logo
+
+![The Strapi Logo](../assets/contribution-guide/contribute-with-docs/strapi-logo.png 'Strapi Logo')
+
+---
+
+#### Use an unordered list for action steps, number the large sections
+
+There are several reasons for using an unordered list for the **actionable** steps in the documentation.
+
+1. Many developers do not read the theory and are just looking for the steps they need for a specific action. Bullets are an effective way to show this.
+2. Using unordered lists ensure all **actionable** steps are consistent throughout the docs and make the experience better for developers.
```

Comment:
What is the source of this affirmation?

Reply:
I have interviewed (unscientifically) developers and researched this a bit. It seems to be "common knowledge" and observable, that a developer returning to docs, will go back to a section and copy/paste the steps or code snippet to accomplish something.

The #1 can be removed as it doesn't change the section and its purpose.  However, it's observable that developers copy/paste from docs. If it's a "big deal" it can safely be removed from this section but it actually adds an additional aspect to why we use unordered lists here.

A quick search in Google finds the following articles that touch upon this behavior:

https://zapier.com/engineering/great-documentation-examples/
https://www.heavybit.com/library/video/how-great-documentation-drives-developer-adoption/
https://www.quora.com/Do-programmers-or-developers-actually-read-whole-documentation-of-how-a-technology-is-implemented-before-they-try-to-build-something